_hackbars/hafifAmudi.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: hackbar
+title: Hackbar Warning for Earth
+author: Hafif Amudi
+---
+
+In open source, we can share our knowledge for free and we can make more relation and channel with another developer.
+
+---
+
+## Our World is in Emergency Situation
```

Comment:
Codacy found an issue: [[heading-style] Headings should use setext](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6282547)